designer/restmodel/src/main/scala/pl/touk/nussknacker/restmodel/definition/package.scala

```diff
@@ -78,44 +78,41 @@ package object definition {
   @JsonCodec(encodeOnly = true) final case class UISourceParameters(sourceId: String, parameters: List[UIParameter])
 
   final case class UINodeEdges(
-      componentId: ComponentInfo,
+      componentId: ComponentId,
       edges: List[EdgeType],
       canChooseNodes: Boolean,
       isForInputDefinition: Boolean
   )
 
   object UINodeEdges {
-    implicit val componentIdEncoder: Encoder[ComponentInfo] = Encoder.encodeString.contramap(_.toString)
+    implicit val componentIdEncoder: Encoder[ComponentId] = Encoder.encodeString.contramap(_.toString)
 
     implicit val encoder: Encoder[UINodeEdges] = deriveConfiguredEncoder
   }
 
   object UIComponentNodeTemplate {
 
     def create(
-        componentInfo: ComponentInfo,
+        componentId: ComponentId,
```

Comment:
doesn't this change warrant an entry in the changelog?

Reply:
(I meant to point to the field in the case class, not the `create` method)

Reply:
You mean in the migration guide? I've added it now. I don't think that the end-user is interested in such changes, so it shouldn't be in the changelog.